src/utils/getGridRowCells.js

```diff
@@ -0,0 +1,7 @@
+const getGridRowCells = timebar =>
+  timebar.find(row => (
+    row.useAsGrid) ||
+    timebar[timebar.length - 1]
```

Comment:
Don't think we want a default - this is a feature users should be able to opt into.